tests/finding_test.py

```diff
@@ -362,9 +394,13 @@ def add_finding_tests_to_suite(suite, jira=False, github=False, block_execution=
     suite.addTest(ProductTest('test_add_product_finding'))
     # TODO add some more findings with different statuses
     suite.addTest(FindingTest('test_list_findings_all'))
-    suite.addTest(FindingTest('test_list_findings_closed'))
-    suite.addTest(FindingTest('test_list_findings_accepted'))
-    suite.addTest(FindingTest('test_list_findings_open'))
+    # The next 3 tests don't work and need to be reworked
+    # suite.addTest(FindingTest('test_list_findings_closed'))
+    # suite.addTest(FindingTest('test_list_findings_accepted'))
+    # suite.addTest(FindingTest('test_list_findings_open'))
```

Comment:
what's wrong with the tests, I can help maybe if needed to fix them.

Reply:
I didn't look what's wrong with them. They got never executed properly. Each of these tests calls `test_list_findings` with a parameter for the suffix of the URL like `finding/open` or `finding/closed`. But in `test_list_findings` this suffix is ignored and it executes the same test for all findings 4 times. Have a look at https://github.com/DefectDojo/django-DefectDojo/blob/master/tests/finding_test.py#L17-L32 and you will see.

It's completely unrelated with this PR, just something I noted when I saw it while writing more tests.

Reply:
It should be fixed in a separate PR

Reply:
ah oke, at first flance it looked like this PR broke. maybe put that comment in there and leave it for another PR indeed.

Reply:
I wrote issue #5154 to address this.